tictactoe/models/Board.java

```diff
@@ -0,0 +1,97 @@
+package tictactoe.tictactoePackage.models;
+
+public class Board {
+
+    private int sizeOfBoard;
+    private Character  board[][];
+
+    public Board(int sizeOfBoard){
+        this.sizeOfBoard = sizeOfBoard;
+        this.board = new Character [sizeOfBoard][sizeOfBoard];
+    }
+
+    public void setPieceAtBoard(int currentRow, int currentColumn, Character  currentPiece) {
+        if(isPieceAtValidCell(currentRow, currentColumn)){
+            board[currentRow][currentColumn] = currentPiece;
+        }
+        else{
+            System.out.println("Invalid Move");
+        }
+    }
+
+    public void initializeTheBoard(){
+        for (int i=0;i<sizeOfBoard;i++){
+            for (int j=0;j<sizeOfBoard;j++){
+                board[i][j] = '-';
+            }
+        }
+    }
+
+    public void printBoard(){
+        for (int i=0;i<sizeOfBoard;i++){
```

Comment:
Are you not using intellij? Spacing issues are there in the entire codebase.

Reply:
I am using intellij. Will take care of spaces in further questions.